src/tools/schema2base.ts

```diff
@@ -29,18 +30,19 @@ export abstract class Schema2Base {
   }
 
   private async processFile(src: string) {
+    Utils.init('result: ${process.cwd()}');
     const outName = this.opts.outfile || this.outputName(path.basename(src));
     const outPath = path.join(this.opts.outdir, outName);
     console.log(outPath);
     if (this.opts.update && fs.existsSync(outPath) && fs.statSync(outPath).mtimeMs > fs.statSync(src).mtimeMs) {
       return;
     }
 
-    const contents = fs.readFileSync(src, 'utf-8');
-    const manifest = await Manifest.parse(contents);
+    const manifest = await Utils.parse(`import '${process.cwd()}/${src}'`);
```

Comment:
I suppose since you put the full path here, the mangled path in `Utils.init()` won't matter (at least until some file you are processing uses one of the $ macros [e.g. `https://$particles`, these are setup using the path sent to `init`]).

Reply:
Ok, I updated it in the utils.init call as well.